x-pack/heartbeat/monitors/browser/synthexec/enrich.go

```diff
@@ -23,19 +23,22 @@ import (
 type enricher func(event *beat.Event, se *SynthEvent) error
 
 type streamEnricher struct {
-	je      *journeyEnricher
-	sFields stdfields.StdMonitorFields
+	je         *journeyEnricher
+	sFields    stdfields.StdMonitorFields
+	checkGroup string
 }
 
 func newStreamEnricher(sFields stdfields.StdMonitorFields) *streamEnricher {
-	return &streamEnricher{sFields: sFields}
+	return &streamEnricher{sFields: sFields, checkGroup: makeUuid()}
```

Comment:
As described in the main issue description, the UUID is now a member of the stream, and changes when each journey ends, removing periods before journey starts where no UUID is present.